.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "research/denoised_smoothing"]
```

Comment:
The submodule seems to be deleted in commit 211e728. Please delete this file also if the submodule is not going to be used.